tests/console/http_srv.pm

```diff
@@ -11,42 +11,37 @@
 # Summary: Simple apache server test
 # Maintainer: Oliver Kurz <okurz@suse.de>
 
+package http_srv;
+use base "services::apache";
 use strict;
 use warnings;
 use base "consoletest";
 use testapi;
 use utils;
 
+sub run_test {
+    my ($self) = @_;
+
+    $self->install_apache();
+    $self->enable_apache();
+    $self->start_apache_service();
+    $self->check_apache_service();
+    $self->check_apache_function();
```

Comment:
doesn't look you would need to rely on object methods but you could rather call these as free functions